addons/escoria-core/game/escoria.gd

```diff
@@ -93,6 +97,7 @@ func _init():
 	self.resource_cache.start()
 	self.save_manager = ESCSaveManager.new()
 	self.inputs_manager = ESCInputsManager.new()
+	connect("request_pause_menu", inputs_manager, "_on_pause_menu_requested")
```

Comment:
This line should go into the _ready function of inputs manager like

`escoria.connect("request_pause_menu", self, "_on_pause_menu_requested")`

Reply:
Inputs Manager has no _ready() function. It is initialized by Escoria.gd in _init() function, but this won't call _ready() function even if I create one in inputs manager. Inputs Manager has to be added to the scene tree to have the _ready() function called.
This is why I connected it in Escoria directly :)

Reply:
I see. Then I'd recommend adding a "registerCore" function in inputs manager and call that like I did with the background and inventory item objects. That way we can keep the private methods.

Reply:
Alright. Will do!

Reply:
Done